spec.bs

```diff
@@ -763,70 +763,258 @@ enrollment and authentication. This extends the
+    (e.g. `a.com` in the above example).
+
+Secondly, a [=Relying Party=] can consider keeping their payment and login
+credentials separate. If doing this, the [=Relying Party=] should only register
+credentials for Secure Payment Confirmation on a subdomain (e.g.
+`https//payment.b.com`), and should keep payment credentials and login
+credentials separate in their database.
 
-    * Assuming the user does consent and verify the transaction, the
-        Relying Party must still be expecting the payment, otherwise the
-        cryptogram is useless.
+<div class="note">
+NOTE: As currently written, the Secure Payment Confirmation specification
+      allows any WebAuthn credential to be used in an SPC authentication.
+      However this is not true in implementations today, which only allow
+      credentials created with the {{AuthenticationExtensionsClientInputs/payment}}
+      extension specified to participate in SPC authentication, and the
+      specification may be updated to reflect that in the future.
+
+      In both implementation and specification today, a credential created
+      with the {{AuthenticationExtensionsClientInputs/payment}} can be used
+      for login, if the Relying Party wishes. This is not expected to change.
+</div>
 
-* The same argument is used against the concern that a third party (e.g., the
-    merchant) provides many of the input values for SPC authentication
-    (e.g., the card icon, name, etc). Whilst this may be confusing to the
-    user if the merchant is malicious, the cryptogram is ultimately useless
-    unless the Relying Party accepts it - and they are able to
-    cryptographically verify what was shown to the user.
+### Payment Attack ### {#sctn-security-payment-attack}
 
-* Allowing \[[Create]] in an iframe; see https://github.com/w3c/webauthn/issues/1336
-    from WebAuthn.
+Instead of attempting to hijack a user account, a malicious third-party may
+also attempt to initiate an unauthorized payment using Secure Payment
+Confirmation credentials (obtained either legitimately or otherwise). Such an
+attack is infeasible:
 
-    * Implementations can address this concern by providing native 
-        payments-focused browser UX during SPC enrollment.
+* When the attacker initiates SPC, the user will be shown an interface by the
+    User Agent that clearly states the transaction details (including the payee
+    and amount). The user is very likely to reject this interface as invalid.
+* If the user does agree to the transaction, and completes the subsequent
+    WebAuthn authentication ceremony, the attacker now has a signed SPC
+    assertion for the [=Relying Party=].
+* However, the [=Relying Party=] is not expecting a transaction, and so will
+    just reject the assertion it was sent!
 
-    * In the absence of such as UX, a browser can require a
-        user gesture as part of initiating the API.
+NOTE: In addition the above, the assertion `challenge` will also be incorrect.
```

Comment:
I think the third bullet and Note can be massaged to cover two exclusive use cases, both interesting:

* If the [=Relying Party=] is not expecting any transaction, it will reject the assertion.
* If the [=Relying Party=] is expecting a transaction, it will detect an unfamiliar `challenge` and reject the assertion.